GitUI/UserControls/FileStatusDiffCalculator.cs

```diff
@@ -4,41 +4,44 @@
 
 namespace GitUI
 {
-    public sealed partial class FileStatusDiffCalculator
+    public sealed partial class FileStatusDiffCalculator(Func<IGitModule> getModule)
     {
-        private readonly Func<IGitModule> _getModule;
-
         // Currently bound revisions etc. Cache so we can reload the view, if AppSettings.ShowDiffForAllParents is changed.
         private FileStatusDiffCalculatorInfo _fileStatusDiffCalculatorInfo = new();
 
         // Default helper functions, can be set
         public Func<ObjectId, string>? DescribeRevision { get; set; }
         public Func<GitRevision, GitRevision>? GetActualRevision { get; set; }
 
-        public FileStatusDiffCalculator(Func<IGitModule> getModule)
-        {
-            _getModule = getModule;
-        }
```

Comment:
Tbh, I'm not excited by the default .ctors (or whatever they are called). In the future, this .ctor will be used for injecting `IServiceProvider` to resolving the current git module.

Reply:
This was a suggestion, less code so I preferred it.
Can revert and disable the suggestion.

Reply:
reverted